changelogs/fragments/7373-lxc-remote-addr-change.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

I'd probably label this as a new feature, since it changes behavior that was there for a long time and I don't want to backport it because of that.
```suggestion
minor_changes:
```